includes/Specials/SpecialManageWiki.php

```diff
@@ -118,6 +118,12 @@ public function showWikiForm( $wiki, $module, $special, $filtered ) {
 			$out->addModuleStyles( [ 'oojs-ui-widgets.styles' ] );
 		}
 
+		$remoteWiki = new RemoteWiki( $wiki );
+
+		if ( $remoteWiki->isLocked() ) {
+			$out->addHTML( Html::errorBox( $this->msg( 'managewiki-mwlocked' )->plain() ) );
```

Comment:
escaped() instead of plain()

Reply:
Also, needs i18n added (non-Miraheze specific i18n)

Reply:
Doesn't there also need to be a default/generic i18n anyway?